distributedsocialnetwork/distributedsocialnetwork/updates.py

```diff
@@ -22,6 +22,10 @@ def get_all_visible_posts():
     author = Author.objects.filter(host=settings.FORMATTED_HOST_NAME)[0]
     get_visible_posts(author.id)
 
+def get_all_github_activity():
+    # Get the github activity for all our local authors.
+    for author in Author.objects.all.filter(origin__icontains=settings.FORMATTED_HOST_NAME):
```

Comment:
Couple of typos here that fail to load the posts. Change to `Author.objects.filter` (remove the all) and switch `origin__icontains=settings.FORMATTED... to `host=settings.FORMATTED...`

Reply:
Thanks! Fixed.